plugins/poll/assets/stylesheets/common/poll.scss

```diff
@@ -299,64 +344,6 @@ div.poll-outer {
       }
     }
 
-    // .poll-buttons-dropdown {
-    //   align-self: stretch;
-    //   position: relative;
-
-    //   .label {
-    //     display: none;
-    //   }
-    //   .widget-dropdown {
-    //     &.closed {
-    //       :not(:first-child) {
-    //         display: none;
-    //       }
-    //     }
-    //     &.opened {
-    //       display: flex;
-    //       flex-direction: column;
-    //       position: relative;
-
-    //       overflow-y: visible;
-    //       .widget-dropdown-body {
-    //         display: block;
-    //         position: absolute;
-    //         z-index: 300;
-    //         overflow-y: visible;
-    //         transform: translate(-44px, 38px);
-    //         .widget-dropdown-item {
-    //           width: 100%;
-    //           padding: 0;
-    //           margin: 0;
-    //           float: left;
-    //           &:hover {
-    //             color: var(--tertiary);
-    //             background-color: var(--primary-low);
-    //           }
-    //           button {
-    //             width: 100%;
-    //             padding: 0.5em;
-    //             display: flex;
-    //             flex-direction: row;
-    //             background-color: var(--secondary);
-    //             &:hover {
-    //               background-color: var(--primary-low);
-    //             }
-    //             border: none;
-    //           }
-    //         }
-    //       }
-    //     }
-
-    //     &-header {
-    //       height: 100%;
-    //       .d-icon {
-    //         margin: 0;
-    //       }
-    //     }
-    //   }
-    // }
-
```

Comment:
Can you remove this please?

Reply:
@jjaffeux I'm not seeing that in my source at all?